pkg/internal/config/config.go

```diff
@@ -31,24 +31,24 @@ import (
 )
 
 const (
-	NetworkAttachmentDefinitionNameParamName  = "networkAttachmentDefinitionName"
-	TrafficGenContainerDiskImageParamName     = "trafficGenContainerDiskImage"
-	TrafficGenTargetNodeNameParamName         = "trafficGenTargetNodeName"
-	TrafficGeneratorPacketsPerSecondParamName = "trafficGeneratorPacketsPerSecond"
-	VMContainerDiskImageParamName             = "vmContainerDiskImage"
-	DPDKNodeLabelSelectorParamName            = "DPDKNodeLabelSelector"
-	TestDurationParamName                     = "testDuration"
-	PortBandwidthGBParamName                  = "portBandwidthGB"
-	VerboseParamName                          = "verbose"
+	NetworkAttachmentDefinitionNameParamName = "networkAttachmentDefinitionName"
+	TrafficGenContainerDiskImageParamName    = "trafficGenContainerDiskImage"
+	TrafficGenTargetNodeNameParamName        = "trafficGenTargetNodeName"
+	TrafficGenPacketsPerSecondParamName      = "trafficGenPacketsPerSecond"
+	VMContainerDiskImageParamName            = "vmContainerDiskImage"
+	DPDKNodeLabelSelectorParamName           = "DPDKNodeLabelSelector"
+	TestDurationParamName                    = "testDuration"
+	PortBandwidthGBParamName                 = "portBandwidthGB"
+	VerboseParamName                         = "verbose"
 )
 
 const (
-	TrafficGenDefaultContainerDiskImage     = "quay.io/kiagnose/kubevirt-dpdk-checkup-traffic-gen:main"
-	TrafficGeneratorPacketsPerSecondDefault = "14m"
-	VMContainerDiskImageDefault             = "quay.io/kiagnose/kubevirt-dpdk-checkup-vm:main"
-	TestDurationDefault                     = 5 * time.Minute
-	PortBandwidthGBDefault                  = 10
-	VerboseDefault                          = false
+	TrafficGenDefaultContainerDiskImage = "quay.io/kiagnose/kubevirt-dpdk-checkup-traffic-gen:main"
+	TrafficGenDefaultPacketsPerSecond   = "8m"
```

Comment:
I think the value change deserved a separate PR

Reply:
Reverted this change.